### ISSUE TITLE
Hyperparameters / learning rate schedule

### ISSUE THREAD

**Issue by frankier** [fshdnc/eb_class#5]
It seems at the moment the learning rate is fairly high and we reach the minimum loss after just a few epochs. It might be worth trying a lower learning rate and a learning rate schedule in case we can get a bit of extra performance.
